### PR TITLE
[player-4080]

### PR DESCRIPTION
**Pull request by aeng7**
-will set multiAudio state to null if we receive less than 2 tracks (effectively hiding the multi audio icon)

PLAYER-4080 was occurring because the video element is throwing an audio change event on a single audio track, which sends out a multi audio changed event with only 1 audio track. This makes our skin show the multi audio icon. This looks to be a native Safari issue. To workaround this, I've modified the skin to set the multi audio state to null if we receive a multi audio changed event with less than 2 tracks.

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -1116,11 +1116,25 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
...
       }
     },
 
+    /**
+     * Checks to see if we have multiple audio tracks
+     * @param multiAudio The multiAudio object that is fetched from the current video
+     * @returns {boolean} True if the provided object contains more than one track, false otherwise
+     */
+    containsMultiAudio: function(multiAudio) {
+      return !!(multiAudio && multiAudio.tracks && multiAudio.tracks.length > 1);
```
Comment (pilievOoyala) [inline js/controller.js]: Can you stick the result in a temp var and return that?

File: js/controller.js [hunk 2]
```diff
@@ -1116,11 +1116,25 @@ OO.plugin('Html5Skin', function(OO, _, $, W) {
...
+          this.state.multiAudio = null;
+        }
+
         this.renderSkin();
       }
     },
 
+    /**
+     * Checks to see if we have multiple audio tracks
+     * @param multiAudio The multiAudio object that is fetched from the current video
```
Comment (pilievOoyala) [inline js/controller.js]: Can you put the type?

File: js/controller.js (hunk 1 above)
Comment (aeng7) [reply js/controller.js]: Added

File: js/controller.js (hunk 2 above)
Comment (aeng7) [reply js/controller.js]: Added

**Comment by pilievOoyala** [comment]
LGTM

**Pull request merged**